### PR TITLE
Testing/bubblefactory

### PR DESCRIPTION
**Pull request by justinvdk**
Little changes in bubble and add tests for bubblefactory.

< testing, layout

### CODE REVIEW

File: src/test/java/nl/tudelft/model/bubble/BubbleTest.java [hunk 1]
```diff
@@ -118,7 +137,12 @@ public void testSplit() throws SlickException {
         when(mockedBubble1.getBounds()).thenReturn(new Rectangle(0, 0, 0, 0));
         when(mockedBubble2.getBounds()).thenReturn(new Rectangle(0, 0, 0, 0));
         when(mockedResources.getBubbleImage1()).thenReturn(mockedImg);
-        
+
+        Random mockedRandom = mock(Random.class);
+        Helpers.setRandom(mockedRandom);
+
```
Comment (cathmer4) [inline src/test/java/nl/tudelft/model/bubble/BubbleTest.java]: You already did this in the setup.

File: src/test/java/nl/tudelft/model/bubble/BubbleBoundsTest.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+@RunWith(Parameterized.class)
+public class BubbleBoundsTest {
+
+    Image mockedImg;
+
+    AbstractBubble bubble1;
+    AbstractBubble bubble2;
+
+    boolean willIntersect;
+
```
Comment (cathmer4) [inline src/test/java/nl/tudelft/model/bubble/BubbleBoundsTest.java]: These can be made private

**Comment by cathmer4** [comment]
The tests look nice! The only thing is, there aren't actually tests for the BubbleFactory, so those should still be added.

**Comment by justinvdk** [comment]
I planned to first do these and rename the branch, which I forgot.
I will add tests for bubblefactory in following commits.

**Comment by justinvdk** [comment]
please consider re-reviewing this PR < testing, review

**Comment by cathmer4** [comment]
Great!

**Pull request merged**